### PR TITLE
add Submit Button

### PR DESCRIPTION
**Pull request by JpBurgarelli**
Issue: #17

<details open> 
  <summary>
    <b>Feature</b>
  </summary>

Submit button 
</details>

<details> 
  <summary>
    <b>Changelog</b>
  </summary>
N/A
</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>
  </summary>

https://github.com/Alecell/octopost/assets/30197026/8af66fdd-0378-42eb-898b-385a284bf454

</details>

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

  - [x] Build working correctly
  - [x] Tests created
  - [x] Issue linked
</details>

<details> 
  <summary>
    <b>Additional info</b>
  </summary>
N/A
</details>

### CODE REVIEW

File: src/components/SubmitButton/SubmitButton.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+
+import SubmitButton from './SubmitButton';
+
+describe('Submit Button component', () => {
+  test('render Submit Button component', () => {
+    render(<SubmitButton />);
+  });
+});
```
Comment (Alecell) [inline src/components/SubmitButton/SubmitButton.spec.tsx:9]: Esse teste n ta testando nada

File: src/components/SubmitButton/SubmitButton.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+
+import SubmitButton from './SubmitButton';
+
+describe('Submit Button component', () => {
+  test('render Submit Button component', () => {
```
Comment (Alecell) [inline src/components/SubmitButton/SubmitButton.spec.tsx:6]: Tem que usar `it` e não `test`.
A primeira palavra imperativa no ingles. No caso seria "renders", ficando "it renders"

File: src/components/SubmitButton/SubmitButton.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+
+import SubmitButton from './SubmitButton';
+
+describe('Submit Button component', () => {
```
Comment (Alecell) [inline src/components/SubmitButton/SubmitButton.spec.tsx:5]: Deixar só "Submit Button"

File: src/components/SubmitButton/SubmitButton.tsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import scss from './SubmitButton.module.scss';
+
+function SubmitButton() {
+  return <button className={scss.botao}>Clique Aqui </button>;
```
Comment (Alecell) [inline src/components/SubmitButton/SubmitButton.tsx:4]: Evitar esse espacinho do lado do fechamento do </button>, logo depois do "Aqui"

**Review by Alecell — CHANGES_REQUESTED** [review]

**Review by aalmeida00 — CHANGES_REQUESTED** [review]
agreed com todos os review do Ale

**Pull request closed without merging**